gitbook-docs/guides-1/mina-proof-pipeline-example/3.-proof-request.md

```diff
@@ -93,6 +93,7 @@ $ python3 scripts/request_tools.py push \
    "status" : "created"
     ...
 }
+```
```

Comment:
Here we have a pure bash fragment and then a pure JSON fragment. Let's split them into separate code blocks ([example](https://github.com/nilfoundation/proof-market-toolchain#proof-market-beta-access)), so both would have syntax highlighting. And we can make the first block `bash` instead of `console`.

Reply:
OK, fixed